packages/serverless-api/src/client.ts

```diff
@@ -688,6 +683,7 @@ export class TwilioServerlessApiClient extends events.EventEmitter {
       limit: this.config.retryLimit || RETRY_LIMIT,
       methods: ['GET', 'POST', 'DELETE'],
       statusCodes: [429],
+      errorCodes: [],
```

Comment:
This is interesting isn't 429 technically an error code?

Reply:
The `errorCodes`refer to this list: `ETIMEDOUT`, `ECONNRESET`, `EADDRINUSE`, `ECONNREFUSED`, `EPIPE`, `ENOTFOUND`, `ENETUNREACH` and `EAI_AGAIN`. The sort of errors that you don't get a status code for, because something broke before you get a response.

Come to think of it, it might be useful to be able to configure which errors/statuses that this library will retry. Only retrying 429s is useful for the CLI side of things, but that isn't the only thing this could be used for. I think this PR fixes our initial issue, but could be improved to serve users of `serverless-api` in general.